demo/admin/intl-update.ts

```diff
@@ -0,0 +1,67 @@
+import { Gitlab } from "@gitbeaker/node";
+import { existsSync, promises, readFileSync } from "fs";
+import { homedir, hostname } from "os";
+import { dirname } from "path";
+
+let gitlabToken = process.env.GITLAB_API_TOKEN;
+if (!gitlabToken) {
+    const gitlabTokenFile = `${homedir()}/.gitlab_token`;
+    if (!existsSync(gitlabTokenFile)) {
+        console.warn(
+            `Environment-Variable GITLAB_API_TOKEN not present and ${gitlabTokenFile} doesn't exist on '${hostname()}'. Please create a personal token at https://gitlab.vivid-planet.com/profile/personal_access_tokens and save it into this file (both on your machine and your virtual machine on the server`,
```

Comment:
This should probably not be here

Reply:
Fixed in d9372c21f78f7371dda4799bccf5898ff9d8dd25